java/code/src/com/redhat/rhn/domain/action/virtualization/BaseVirtualizationGuestAction.java

```diff
@@ -54,4 +55,12 @@ public void extractParameters(Map context) {
         // therefore does nothing.
     }
 
+    @Override
+    public String getWebSocketActionId() {
+        String id = "new-" + getId();
```

Comment:
Can we use a more specific prefix? Something like `guest-`

Reply:
that one is for now objects... not only guests